test/inductor/test_pattern_matcher.py

```diff
@@ -84,6 +84,58 @@ def fn(a, b, c):
             self.assertEqual(counters["inductor"]["pattern_matcher_count"], 2)
             self.assertEqual(counters["inductor"]["pattern_matcher_nodes"], 4)
 
+    def test_addmm_activation(self):
+        def fn_addmm_relu(input, mat1, mat2):
+            return torch.nn.functional.relu(torch.addmm(input, mat1, mat2))
+
+        def fn_addmm_gelu(input, mat1, mat2):
+            return torch.nn.functional.gelu(torch.addmm(input, mat1, mat2))
+
+        args = [
+            torch.randn(20, device="cuda"),  # input
+            torch.randn(10, 15, device="cuda"),  # mat1
+            torch.randn(15, 20, device="cuda"),  # mat2
+        ]
+
+        for fn, matcher_nodes, atol in (
+            (fn_addmm_relu, 2, 1e-8),
+            # higher tolerance due to the "tanh" approximation
+            # in fused GELU epilogue vs. "none" without fusion
+            (fn_addmm_gelu, 6, 1e-3),
+        ):
+            counters.clear()
+            expected = fn(*args)
+            actual = torch.compile(fn)(*args)
+            torch.testing.assert_close(actual, expected, atol=atol, rtol=0)
+            self.assertEqual(counters["inductor"]["pattern_matcher_count"], 1)
```

Comment:
Can also test that it is hit with the `run_and_get_code` api, and then grep for "addmm_activation"